scripts/ImageBinary.py

```diff
@@ -0,0 +1,105 @@
+###SOM check to ensure SOMs are being printed correctly.
```

Comment:
If the script is only for testing, it should be moved to test directory.

Reply:
Moved